src/config.rs

```diff
@@ -148,6 +158,17 @@ fn extract_default_features(cli_metadata: &Map<String, Value>) -> anyhow::Result
     }
 }
 
+fn extract_rustflags(cli_metadata: &Map<String, Value>) -> Option<Vec<String>> {
+    let rustflags = cli_metadata.get("rustflags")?;
+    match rustflags {
+        Value::Array(features) => features
+            .iter()
+            .map(|value| value.as_str().map(std::string::ToString::to_string))
```

Comment:
I think it would be nice if we could emit an error when no strings are provided instead of silently ignoring it, similar to the feature extraction